Sklep/MainWindow.cs

```diff
@@ -2,24 +2,70 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Data;
+using System.Diagnostics.Contracts;
 using System.Drawing;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 using System.Windows.Forms;
+using AForge.Video;
+using AForge.Imaging;
+using AForge.Math;
+using AForge.Video.DirectShow;
 using Sklep.Database;
+using Sklep.Utils;
+using ZXing;
+using AForge.Imaging.Filters;
+using static System.Windows.Forms.VisualStyles.VisualStyleElement;
 
 namespace Sklep
 {
     public partial class MainWindow : Form
     {
+        BarcodeScanner barcodeScanner = new BarcodeScanner();
         public MainWindow()
         {
             InitializeComponent();
 
             // TODO: Remove this sample code as it's just for tests
             DatabaseConnection db = new DatabaseConnection();
             db.Connect(DatabaseConnectionSettings.FromEnv());
+            barcodeScanner.pictureBox = pictureBox1;
+            barcodeScanner.pictureBox.SizeMode = PictureBoxSizeMode.StretchImage;
+            barcodeScanner.CodeScanned += BarcodeScanner_CodeScanned;
+            barcodeScanner.startScanning();
+
+        }
+
+        private void BarcodeScanner_CodeScanned(object sender, string code)
+        {
+            EANValidator validator = new EANValidator();
```

Comment:
Walidację wrzuciłbym do BarcodeScannera, bo teraz jeśli w 3 miejscach będziemy skanować kody, to w 3 miejscach trzeba będzie powtarzać walidowanie